imap_processing/tests/spice/test_kernels.py

```diff
@@ -1,33 +1,65 @@
 """Tests coverage for imap_processing/spice/kernels.py"""
 
+import numpy as np
 import pytest
 import spiceypy as spice
 from spiceypy.utils.exceptions import SpiceyError
 
-from imap_processing.spice import kernels
+from imap_processing.spice.kernels import (
+    _average_quaternions,
+    _create_rotation_matrix,
+    _get_et_times,
+    create_pointing_frame,
+    ensure_spice,
+)
+
+
+@pytest.fixture()
+def kernels(spice_test_data_path):
+    """List SPICE kernels."""
+    # TODO: remove this fixture after ensure_spice update.
```

Comment:
I'm not sure this TODO is real. We may add a test that checks the `ensure_spice` functionality when wrapping the `create_pointing_frame` function, but I don't anticipate changes that affect how kernels are furnished for tests.